addons/website_forum/static/src/interactions/website_forum_spam.js

```diff
@@ -0,0 +1,66 @@
+import { browser } from "@web/core/browser/browser";
+import { registry } from "@web/core/registry";
+import { KeepLast } from "@web/core/utils/concurrency";
+import { renderToElement } from "@web/core/utils/render";
+import { Interaction } from "@web/public/interaction";
+import { cloneContentEls } from "@website/js/utils";
+
+class WebsiteForumSpam extends Interaction {
+    static selector = ".o_wforum_moderation_queue";
+    dynamicContent = {
+        ".o_wforum_select_all_spam": { "t-on-click": this.onSelectAllSpamClick },
+        ".o_wforum_mark_spam": { "t-on-click": this.blockedUntilDone(this.onMarkSpamClick) },
```

Comment:
Here you can write `this.blockedUntilDone(this.onMarkSpamClick, true)` since the button had an animation when processing because of the keyword `async`